### DISCUSSION TITLE
This user account is blocked.

### DISCUSSION THREAD

**Discussion by XiaoyaoGG1024** [vercel/community#3774] [Account Issue] [answered]
### Question

Hello, we have contacted you via email with my email processing number 00153641. I am now contacting you through Github and would like to help me unblock this account. Thank you

**Comment by XiaoyaoGG1024** [comment]
help me

**Comment by XiaoyaoGG1024** [comment]
@amyegan

**Reply by sotocodev** [reply]
Please help me, I have the same problem.

**Reply by XiaoyaoGG1024** [reply]
They haven't replied to me yet, I haven't solved it myself

**Reply by amyegan** [reply, marked as answer]
Sending the email was the right thing to do. Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. 🙇 

@XiaoyaoGG1024, it looks like your ticket received a response on Friday. Please check your spam folder in case their response was blocked. They are waiting for you to reply using the email associated with your git username so they can verify your account.

**Reply by sotocodev** [reply]
@amyegan I have not received anything

**Reply by amyegan** [reply]
@Jesussotoweblink If you share the case number, I can check on it for you

**Reply by sotocodev** [reply]
@amyegan This is 00154040

**Reply by sotocodev** [reply]
@amyegan why is this problem?

**Reply by nataligavryliak** [reply]
My case number is 00153467! Please look for me as well

**Comment by jimmythefung** [comment]
I am experiencing the same issue.

**Reply by XiaoyaoGG1024** [reply]
They haven't replied to me yet, I haven't solved it myself

**Comment by nataligavryliak** [comment]
The same problem!!!! What's going on!? It seems that the problem is massive!

**Comment by swarnava** [comment]
Closing the thread. For posterity, if your account is blocked during registration, please email at registration@vercel.com
